api/actions/actionsRouter.js

```diff
@@ -1,8 +1,116 @@
 const express = require('express');
-const { validateSubjectBody } = require('./actionsMiddleware');
-const Actions = require('./actionsModel');
 const router = express.Router();
+const Actions = require('./actionsModel');
+const { validateSubjectBody } = require('./actionsMiddleware');
+/**
+ * @swagger
+ * components:
+ *  schemas:
+ *    Actions:
```

Comment:
"Actions" should be singular; this is what a _single_ action should look like.

Reply:
Fixed it